src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.gpiod_chip_get_line.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.Device.Gpio;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    [DllImport(library, SetLastError = true)]
+    internal static extern SafeLineHandle gpiod_chip_get_line(SafeChipHandle chip, int offset);
```

Comment:
Where did you get the method declaraions from? It would help if you provide a link to the header file on this documents so that we know how to upgrade it if necessary.

Reply:
The class name is matching the method name we are calling, I thought that is the convention